vendor/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
do we need this file?

Reply:
no, we don't. it is coming because of  indirect dependency in this vendor file `k8s.io/kubernetes/pkg/features`